src/app/ui/gatekeeper/Gatekeeper.style.ts

```
@@ -0,0 +1,41 @@
+import { ButtonPrimary } from '@/components/Button/Button.style';
+import Box from '@mui/material/Box';
+import { styled } from '@mui/material/styles';
+import { Link } from '@/components/Link';
+
+export const GatekeeperIllustrationWrapper = styled(Box)(({}) => ({
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)